packages/client/hmi-client/src/services/charts.ts

```diff
@@ -1323,21 +1323,52 @@ export function createSimulateSensitivityScatter(samplingLayer: SensitivityChart
 /* -------------------------------------------------------------------------- */
 
 export function createSensitivityRankingChart(data: Map<string, number>, options: BaseChartOptions) {
+	// use only 20 highest scores
+	const parametersShownCount = 20;
+	const foramttedData = Array.from(data)
+		.map(([parameter, score]) => ({ parameter, score }))
+		.sort((a, b) => Math.abs(b.score) - Math.abs(a.score))
+		.slice(0, parametersShownCount);
+
+	let title = '';
+	if (foramttedData.length === parametersShownCount) {
+		title = `Top ${parametersShownCount} most sensitive parameters displayed.`;
+	}
+	const notShownCount = data.size - foramttedData.length;
+	if (title && notShownCount > 0) title += ` ${notShownCount} parameters not shown.`;
+
+	const globalFont = 'Figtree';
```

Comment:
@jryu01 : This is a bit silly, that we repeated declare a globalFont at a local scope, maybe we can mark this as cleanup todo?

Reply:
yeah I think we should